### DISCUSSION TITLE
[Blog post] Welcoming Brad Cornes to the Tailwind Team!

### DISCUSSION THREAD

**Discussion by adamwathan** [tailwindlabs/tailwindcss#2040] [Show and tell]
Last month, Brad Cornes joined the team as employee #1! We are super excited to have him on board, and I wrote about what it's been like working with him so far and how he's been helping us out a bit today:

[Read "Welcoming Brad Cornes to the Tailwind Team"](https://blog.tailwindcss.com/welcoming-brad-cornes-to-the-tailwind-team)

Please give him a warm welcome! 😍 👏

**Comment by Apostolique** [comment]
Awesome! Great job on the Tailwind CSS IntelliSense plugin. It's gotten so much better in the past couple months!
